accounts/models.py

```diff
@@ -30,10 +32,12 @@ def create_superuser(self, email, password, **kwargs):
 
 class User(AbstractBaseUser, PermissionsMixin):
     USERNAME_FIELD = 'email'
-
-    email = models.EmailField(unique=True)
+    display_name = models.TextField(unique=True, null=True, max_length=30)
+    email = models.EmailField(unique=True, max_length=75)
```

Comment:
For usernames you don't want anything over 30.   I believe the email one was already set to 75, but don't care if it is 254 or what not.